src/applications/disability-benefits/526EZ/pages/veteranInfo.js

```diff
@@ -1,64 +1,31 @@
-import { merge } from 'lodash/fp';
+import AsyncDisplayWidget from '../components/AsyncDisplayWidget';
 
-import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
-
-import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
-import fullNameUI from '../../../common/schemaform/definitions/fullName';
-import ssnUI from '../../../common/schemaform/definitions/ssn';
-import { genderLabels } from '../../../../platform/static-data/labels';
-
-import ReviewCardField from '../components/ReviewCardField';
-
-import {
-  veteranInformationViewField,
-  VAFileNumberDescription
-} from '../helpers';
+import { veteranInformationViewField } from '../helpers';
 
 export const uiSchema = {
-  'ui:field': ReviewCardField,
+  'ui:field': 'StringField',
+  'ui:widget': AsyncDisplayWidget,
```

Comment:
This looks good to me, but it makes me think that maybe there's a use case for some kind of hook in between a fully custom page component and the customization options in RJSF (fields, widgets, etc). Or to just adjust how the custom page component works. It doesn't look like you actually need anything RJSF related in here, but making a custom page component the way it works now would mean too much duplication.

Reply:
Right, exactly. I'm not sure what the solution to the duplication problem is, but a custom page seems like it might be a better fit. This doesn't feel _too_ wonky, though. :man_shrugging: